### ISSUE TITLE
Overly long text input seems to blow up the application

### ISSUE THREAD

**Issue by StasTserk** [ngrichyj4/askr.xyz#8]
If we put in exessively long text into the option field the application will generate an error instead of showing the voting page.

This should be either fixed by updating the option validation code or by accomodating for longer option text on the backend

**Comment by tamcgoey** [comment]
Good point. Maybe @ngrichyj4 or @alexanderbird could look into some model validation.
